lib/src/select/Select.stories.tsx

```diff
@@ -0,0 +1,536 @@
+      </ExampleContainer>
+      <ExampleContainer>
+        <Title title="Medium size" theme="light" level={4} />
+        <DxcSelect label="Medium" options={single_options} size="medium" />
+      </ExampleContainer>
+      <ExampleContainer>
+        <Title title="Large size" theme="light" level={4} />
+        <DxcSelect label="Large" options={single_options} size="large" />
+      </ExampleContainer>
+      <ExampleContainer>
+        <Title title="Fillparent size" theme="light" level={4} />
+        <DxcSelect label="Fillparent" options={single_options} size="fillParent" />
+      </ExampleContainer>
+    </>
+    <>
+      <Title title="Margins" theme="light" level={2} />
+      <ExampleContainer>
+        <Title title="xxsmall margin" theme="light" level={4} />
+        <DxcSelect label="xxSmall" options={single_options} margin="xxsmall" />
+      </ExampleContainer>
+      <ExampleContainer>
+        <Title title="xsmall margin" theme="light" level={4} />
+        <DxcSelect label="xSmall" options={single_options} margin="xsmall" />
+      </ExampleContainer>
+      <ExampleContainer>
+        <Title title="small margin" theme="light" level={4} />
+        <DxcSelect label="Small" options={single_options} margin="small" />
+      </ExampleContainer>
+      <ExampleContainer>
+        <Title title="medium margin" theme="light" level={4} />
+        <DxcSelect label="Medium" options={single_options} margin="medium" />
+      </ExampleContainer>
+      <ExampleContainer>
+        <Title title="large margin" theme="light" level={4} />
+        <DxcSelect label="Large" options={single_options} margin="large" />
+      </ExampleContainer>
+      <ExampleContainer>
+        <Title title="xlarge margin" theme="light" level={4} />
+        <DxcSelect label="xLarge" options={single_options} margin="xlarge" />
+      </ExampleContainer>
+      <ExampleContainer>
+        <Title title="xxlarge margin" theme="light" level={4} />
+        <DxcSelect label="xxLarge" options={single_options} margin="xxlarge" />
+      </ExampleContainer>
+    </>
+    <ExampleContainer>
+      <Title title="Ellipsis" theme="light" level={4} />
+      <Title title="Multiple selection with ellipsis" theme="light" level={4} />
+      <DxcSelect label="Label" options={single_options} multiple value={["1", "2", "3", "4"]} />
+      <Title title="Value with ellipsis" theme="light" level={4} />
+      <DxcSelect label="Label" options={optionsWithEllipsisMedium} value="1" size="medium" />
+      <Title title="Option with ellipsis" theme="light" level={4} />
+      <DxcSelect label="Label" options={optionsWithEllipsisMedium} value="1" size="medium" />
+    </ExampleContainer>
+  </>
+);
+export const Chromatic = Select.bind({});
+Chromatic.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement);
+  await userEvent.click(canvas.getAllByRole("combobox")[25]);
+};
+const DefaultSelect = () => (
```

Comment:
Can we use this test to check the scenario of a select marked as optional (which I think remains)? 

![image](https://user-images.githubusercontent.com/44321109/153152903-0960a6a6-9851-4f00-903e-3d8b88ed22c4.png)

Reply:
Done